subsys/emds/Kconfig

```diff
@@ -55,9 +60,9 @@ config EMDS_FLASH_TIME_ENTRY_OVERHEAD_US
 
 config EMDS_FLASH_TIME_BASE_OVERHEAD_US
 	int "Time to schedule the store process"
-	default 18200 if SETTINGS && SOC_FLASH_NRF_RRAM
-	default 85000 if SETTINGS && !SOC_FLASH_NRF_PARTIAL_ERASE
-	default 9000 if SETTINGS && SOC_FLASH_NRF_PARTIAL_ERASE
+	default 1000 if SETTINGS && SOC_FLASH_NRF_RRAM
+	default 85000 if SETTINGS && !SOC_FLASH_NRF_PARTIAL_ERASE && !SOC_FLASH_NRF_RRAM
```

Comment:
Is the `&& !SOC_FLASH_NRF_RRAM` needed here? Wouldn't this always be false when we get here, since otherwise it would have picked the first default?